zetaclient/zetacore/client_monitor.go

```diff
@@ -81,7 +81,7 @@ func (c *Client) monitorVoteOutboundResult(
 			}
 		}
 	default:
-		c.logger.Debug().Fields(logFields).Msg("monitorVoteOutboundResult: successful")
+		c.logger.Info().Fields(logFields).Msg("monitorVoteOutboundResult: successful")
```

Comment:
Changes here will just make easier to find the inbound processing tx from Datadog